README.md

```diff
@@ -350,14 +368,22 @@ The only reserved ability MUST be the un-namespaced [`"*"` or "superuser"](#41-s
 
 ### 3.2.5 Proof of Delegation
 
-The `prf` field MUST contain UCAN witnesses (the "inputs" of a UCAN). As they need to be independently verifiable, proofs MUST either be the fully encoded version of a UCAN (including the signature) or the content address of the relevant proof. Attenuations not covered by a proof in the `prf` array MUST be treated as owned by the issuer DID.
+The `prf` field MUST contain the content address <!-- FIXME add link to CID discussion later in doc? --> of UCAN proofs (the "inputs" of a UCAN). Attenuations not covered by a proof in the `prf` array MUST be treated as <!-- FIXME as invalid, because we now have my: and as: --> owned by the issuer DID.
```

Comment:
Can you still inline proofs? Or do you _always_ use the CID & send the proof along as supporting evidence

Reply:
The idea is to always use CIDs as the canonical representation (the thing you sign) so that you don't have to send the entire DAG every time, and can decouple the collection representation from the token. e.g. CAR and just straight-up JSON